hack/build.sh

```diff
@@ -15,5 +15,6 @@
 # limitations under the License.
 
 set -x
-rm -f kf
-go build ./cmd/kf/...
+mkdir -p bin
+rm -f bin/*
```

Comment:
from a safety perspective I'm not really a fan of this, I can see someone running hack/build.sh from their a different directory which could be a mess

Reply:
the rm wasn't necessary so I, ahem, rm'd it